apps/etterlatte-behandling/src/main/kotlin/grunnlagsendring/BeroertePersonerVedDoedsfallService.kt

```diff
@@ -0,0 +1,36 @@
+package no.nav.etterlatte.grunnlagsendring
+
+import no.nav.etterlatte.common.klienter.PdlTjenesterKlient
+import no.nav.etterlatte.libs.common.behandling.SakType
+import no.nav.etterlatte.libs.common.pdl.PersonDTO
+import no.nav.etterlatte.libs.common.person.Person
+import no.nav.etterlatte.libs.common.person.PersonRolle
+import java.time.LocalDate
+import java.time.temporal.ChronoUnit
+
+class BeroertePersonerVedDoedsfallService(private val pdlTjenesterKlient: PdlTjenesterKlient) {
+    // Finner beroerte personer basert på regler beskrevet i https://confluence.adeo.no/display/TE/Funksjonelle+krav
+    fun hentBeroertePersoner(fnr: String): List<Person> {
+        val avdoed = pdlTjenesterKlient.hentPdlModell(fnr, PersonRolle.AVDOED, SakType.BARNEPENSJON)
+
+        assert(avdoed.doedsdato != null) { "Personen er ikke død" }
```

Comment:
Ville droppet `assert` til fordel for `checkNotNull` eller `requireNotNull`. 

`assert` fordrer at vi har aktivert flagget i JVM-en, noe jeg er litt usikker på om vi har.

Reply:
Enig!